src/custom/pages/Profile/index.tsx

```diff
@@ -499,7 +499,7 @@ export default function Profile() {
               </FlexWrap>
             </ChildWrapper>
           </GridWrap>
-          {!account && <Web3Status openOrdersPanel={() => console.log('TODO')} />}
+          {!account && <Web3Status />}
```

Comment:
be careful with this if there is async logic setting state or whatever inside Web3Status as you'll get memory leaks. Could be best to render conditoinally inside `Web3Status`

Reply:
When you get to review https://github.com/cowprotocol/cowswap/pull/596 please double check if this is still a concern, and whether it should be fixed.